dev.lynith.Speedtest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "dev.lynith.Speedtest",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "speedtest",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--persist=.local/share/speedtest/"
```

Comment:
now are you sure it downloads it in this directory instead of using the XDG standard?